fake-server/index.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express')
+const bodyParser = require("body-parser");
+const app = express();
+const port = 3000
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.post('/', (req, res) => {
+    console.log(req.body);
+    const { username, password } = req.body || {};
+    if (username !== "james") {

```

Comment:
Please perform validation for password too, that way I know both username and password are correct in the POST request I send to the server

Reply:
ok i switched the logic order around and added password in